apps/login/src/components/set-password-form.tsx

```diff
@@ -192,34 +192,42 @@ export function SetPasswordForm({
     <form className="w-full">
       <div className="pt-4 grid grid-cols-1 gap-4 mb-4">
         {codeRequired && (
-          <div className="flex flex-row items-end">
-            <div className="flex-1">
-              <TextInput
-                type="text"
-                required
-                {...register("code", {
-                  required: "This field is required",
-                })}
-                label="Code"
-                autoComplete="one-time-code"
-                error={errors.code?.message as string}
-                data-testid="code-text-input"
-              />
-            </div>
-
-            <div className="ml-4 mb-1">
-              <Button
-                variant={ButtonVariants.Secondary}
-                data-testid="resend-button"
-                onClick={() => resendCode()}
+          <Alert type={AlertType.INFO}>
+            <div className="flex flex-row">
+              <span className="flex-1 mr-auto text-left">
+                {t("set.noCodeReceived")}
+              </span>
+              <button
+                aria-label="Resend OTP Code"
                 disabled={loading}
+                type="button"
+                className="ml-4 text-primary-light-500 dark:text-primary-dark-500 hover:dark:text-primary-dark-400 hover:text-primary-light-400 cursor-pointer disabled:cursor-default disabled:text-gray-400 dark:disabled:text-gray-700"
```

Comment:
Not to fix now but I think we should start utilising `gap` more if we're using flexbox anyway. These rogue margins can get quite annoying to maintain eventually

Reply:
Yeah makes sense. I'll think about changing this whenever it comes up again 👍